### ISSUE TITLE
[BUG] Can't load scenario from config file

### ISSUE THREAD

**Issue by yueshuaing** [wsp-sag/network_wrangler#389]
## Describe the bug
I created scenario from scenario by applying project cards in different folders, and saved it using Scenario.write(). When I tried to load the saved scenario using the config file, the process failed. The exported configuration file:
1. Contains some anchors, alias, and paths which seem not to be correct.
2. The scenario is nested within another scenario and the project cards were also written to the file.
3. It inlcudes `_planned_projects` and `_queued_projects` which can't be read
```python
TypeError: ScenarioInputConfig.__init__() got an unexpected keyword argument '_planned_projects'
```

```python
applied_projects: &id001
- test managed lane project
- 6th street transitway
- widen minnehaha pkwy
- 6th st e road diet
base_scenario:
  _planned_projects: []
  _queued_projects: null
  applied_projects: *id001
  base_scenario:
    roadway:
      dir: \\dcclda00dat02.corp.pbwan.net\sag\projects\Met_Council\Yue_temp\metcouncil\network_wrangler\examples\stpaul
      file_format: geojson
    transit:
      dir: \\dcclda00dat02.corp.pbwan.net\sag\projects\Met_Council\Yue_temp\metcouncil\network_wrangler\examples\stpaul
  conflicts: &id003
    6th st e road diet: []
  corequisites: &id004
    6th st e road diet: []
  name: '20241011125709'
  prerequisites: &id005
    6th st e road diet: []
  project_cards:
    6th street transitway: !!python/object:projectcard.projectcard.ProjectCard
      _sub_projects: []
      dependencies: {}
      file: !!python/object/apply:pathlib.WindowsPath
      - \\dcclda00dat02.corp.pbwan.net\sag\
      - projects
      - Met_Council
      - Yue_temp
      - metcouncil
      - network_wrangler
      - examples
      - stpaul
      - project_cards
      - road.prop_change.multiple.yml
      notes: Transitions 6th St E to transitway
      project: 6th Street Transitway
      roadway_property_change:
        facility:
          from:
            osm_node_id: '187899923'
          links:
            ignore_missing: true
            name:
            - 6th
            - Sixth
            - sixth
```

## To Reproduce
Run `pytest tests\test_scenario.py`
The function to test is `test_load_scenario_from_config`

**Comment by e-lo** [comment]
Got it. Fixed issue and pushing change momentarily.  I did consolidate this test with another b/c this test depended on another one being run and we can't have dependency between the tests b/c when they are run in github actions they are distributed between various runners.

**Comment by e-lo** [comment]
fixed in [a71084f](https://github.com/wsp-sag/network_wrangler/commit/a71084f4e135d943bd314ad72dd81e5ba3c70f02)
